components/benefit_sponsors/app/controllers/benefit_sponsors/inboxes/messages_controller.rb

```diff
@@ -74,7 +73,7 @@ def set_sent_box
       end
 
       def is_broker?
-        return (@inbox_provider.class.to_s == "Person") && (/.*BrokerAgencyProfile$/.match(@inbox_provider.broker_role.broker_agency_profile._type))
+        (@inbox_provider.class.to_s == "Person") && /.*BrokerAgencyProfile$/.match(@inbox_provider.broker_role.broker_agency_profile._type)
```

Comment:
It might be good to add a safe nav operator to the second conditional, just in case the `@inbox_provider` is a `Person` w/o a broker role, but it's definitely not essential.